configs/milvus.yaml

```diff
@@ -326,6 +326,9 @@ proxy:
     port:  # high-level restful api
     acceptTypeAllowInt64: true # high-level restful api, whether http client can deal with int64
     enablePprof: true # Whether to enable pprof middleware on the metrics port
+    enableHSTS: true # Enables HTTP Strict Transport Security (HSTS) header support
```

Comment:
Please also change this to default to false.
If the user chooses to use the latest configuration file of Milvus directly, they can decide whether to enable it.
Currently, our configuration logic is that if there is a configuration item, the code default value will not be used.
If this is configured as true, using the latest configuration will enable HSTS by default.